yarn/src/test/scala/org/apache/spark/network/yarn/YarnShuffleServiceSuite.scala

```diff
@@ -87,8 +87,10 @@ class YarnShuffleServiceSuite extends SparkFunSuite with Matchers with BeforeAnd
 
     val execStateFile = s1.registeredExecutorFile
     execStateFile should not be (null)
-    val shuffleInfo1 = new ExecutorShuffleInfo(Array("/foo", "/bar"), 3, "sort")
-    val shuffleInfo2 = new ExecutorShuffleInfo(Array("/bippy"), 5, "hash")
+    val shuffleInfo1 = new ExecutorShuffleInfo(Array("/foo", "/bar"), 3,
+      "org.apache.spark.shuffle.sort.SortShuffleManager")
```

Comment:
Since you did it for other tests, you could have a constant here for the name too.